src/pages/settings/ProfilePage/LoginField.js

```diff
@@ -0,0 +1,101 @@
+import Navigation from '../../../libs/Navigation/Navigation';
+import {resendValidateCode} from '../../../libs/actions/User';
+
+const propTypes = {
+    // Label to display on login form
+    label: PropTypes.string.isRequired,
+
+    // Type associated with the login
+    type: PropTypes.oneOf([CONST.LOGIN_TYPE.EMAIL, CONST.LOGIN_TYPE.PHONE]).isRequired,
+
+    // Login associated with the user
+    login: PropTypes.shape({
+        partnerUserID: PropTypes.string,
+        validatedDate: PropTypes.string,
+    }).isRequired,
+};
+const LoginField = ({
+    label,
+    login,
+    type,
+}) => {
+    let note;
+    if (type === CONST.LOGIN_TYPE.PHONE) {
+        // No phone number
+        if (!login.partnerUserID) {
+            note = 'Add your phone number to settle up via Venmo.';
+
+            // Has unvalidated phone number
+        } else if (!login.validatedDate) {
+            note = 'The number has not yet been validated. Click the button to resend the validation link via text.';
+
+            // Has verified phone number
+        } else {
+            note = 'Use your phone number to settle up via Venmo.';
+        }
+
+        // Has unvalidated email
+    } else if (login.partnerUserID && !login.validatedDate) {
+        note = 'The email has not yet been validated. Click the button to resend the validation link via text.';
+    }
+
+    return (
+        <View style={styles.mb6}>
+            <Text style={styles.formLabel}>{label}</Text>
+            {!login.partnerUserID ? (
+                <Pressable
+                    style={[styles.createMenuItem, styles.ph0]}
+                    onPress={() => Navigation.navigate(type === CONST.LOGIN_TYPE.PHONE
+                        ? ROUTES.SETTINGS_ADD_PHONE
+                        : ROUTES.SETTINGS_ADD_EMAIL)}
+                >
+                    <View style={styles.flexRow}>
+                        <View style={styles.createMenuIcon}>
+                            <Icon src={Plus} />
+                        </View>
+                        <View style={styles.justifyContentCenter}>
+                            <Text style={[styles.createMenuText, styles.ml3]}>
+                                {`Add ${label}`}
+                            </Text>
+                        </View>
+                    </View>
+                </Pressable>
+            ) : (
+                <View style={[styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter]}>
+                    <Text style={[styles.textP]} numberOfLines={1}>
+                        {login.partnerUserID}
+                    </Text>
+                    {!login.validatedDate && (
+                        <Pressable
+                            style={styles.button}
+                            onPress={() => resendValidateCode(login.partnerUserID)}
+                        >
+                            <Text style={styles.createMenuText}>
+                                Resend
```

Comment:
Not sure how feasible this is (and cc: @michelle-thompson or @shawnborton for your opinions on this) but I was really thrown by the lack of feedback when pushing this button. 

Since there's no alternate option like navigating to a new page or clearing the form to show that we've done something, it's pretty hard to tell if the Resend button has worked correctly. 

Trying to think of easy ways to provide even just a little bit of feedback, but I'm coming up short.

Reply:
Here's a video showing what it looks like now: 

https://user-images.githubusercontent.com/16747822/113212752-0d0b6900-922c-11eb-8f08-9915d1b9121c.mov



You can see we do infact queue the notification, so it's working correctly... but you'd never know until your checked your texts or email 

```SQL
sqlite> select * from notifications;
    json = {"accountEmail":"+12345678903@expensify.sms","accountID":41,"authToken":"xyz","isFromExpensifyCash":true,"template":"SecondaryLoginAdd","to":"nikki@meepmeep.com"}
priority = 200
 created = 2021-03-31 21:16:54

Reply:
Maybe we can change the `Resend` text to a checkmark to confirm it was sent.
![image](https://user-images.githubusercontent.com/43361905/113218715-d9810c80-9234-11eb-88f3-e82783126ca6.png)

Reply:
This looks good to me!